decentraland/unity-explorer
# Fix: multiplayer self replica via remote profile flow

## What does this PR change?

- Changed the creation flow for self-replica to use Remote Profiles logic. As result, playing emotes now also reflected by self-replica.
- handled scene emotes (works for the players inside the scene)


## How to test the changes?

verify self-replica (movement doesn't work currently)

1. Launch the explorer
2. Go to /debug-> multiplayer movement
3. Press "Instantiate Self-replica" button -> verify random avatar appeared at your place
5. Play emote and verify that your replica starts emote as well
6. Press "Remove self-replica" -> verify this avatar disappears
7. Press again "Instantiate Self-replica" button -> verify random avatar appeared at your place, it replicates your emotes 

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
